src/binHandler.cpp

```diff
@@ -35,7 +35,7 @@ int dumpBinFiles(std::vector<std::string> &binVect, const std::string outFn) {
 	size_t arrBytes = 0, fileBytes = 0, totalBytes = 0;
 	
 	//Create a heap byte array (From defined size in header)
-	char *byteArray = new char[_def_ARR_SIZE];
+	std::vector<char> byteArray(_def_ARR_SIZE);
```

Comment:
This was intentionally an array to improve speed on older systems, as vectors are heavier than arrays and the nature of the implementation increases machine code by a order of magnitude - The class/object concepts are a detriment to speed here. Please see the usecase notes in README.md. This is intended for anything down to a pentium class CPU for people without access to decent computers

Reply:
In response to this, the compiler should optimize it to more or less the same code for the accesses, but if you look at my changes from fstream's get to the read call to read in blocks it makes it really just used as an raii container like a smart pointer, but with more functionality.
And I would ask you to consider (if binary generation is truly an issue) using std::string instead as you already use it so that you can avoid the risk of leaking memory from an exception being thrown somewhere in the code between the new, and deletes that aren't managed by raii containers.